qos-monitor/src/main/java/eu/arrowhead/core/qos/QoSMonitorController.java

```diff
@@ -683,4 +712,49 @@ private void validateQoSBestRelayRequest(final QoSBestRelayRequestDTO request, f
 			throw new BadPayloadException("attribute is null or empty", HttpStatus.SC_BAD_REQUEST, origin);
 		}
 	}
+
+	//-------------------------------------------------------------------------------------------------
+	private void validateEvent(final EventDTO request, final String origin) {
+		logger.debug("validateEvent started...");
+
+		if (request == null) {
+			throw new BadPayloadException("QoSBestRelayRequestDTO is null", HttpStatus.SC_BAD_REQUEST, origin);
+		}
+
+		if (Utilities.isEmpty(request.getEventType())) {
+			throw new BadPayloadException("EventType is null or empty", HttpStatus.SC_BAD_REQUEST, origin);
+		}
+
+		if (!checkExternalPingMonitoringNotificationEventType(request.getEventType())) {
+			throw new BadPayloadException("EventType is null or empty", HttpStatus.SC_BAD_REQUEST, origin);
```

Comment:
I think the error message here is incorrect (probably from a copy-paste)

Reply:
fixed